src/iccpd/src/cmd_option.c

```diff
@@ -162,7 +169,11 @@ void cmd_option_parser_dump_usage(struct CmdOptionParser* parser, char* prog_nam
         begin = 0;
         length = 0;
         first_line = 1;
-        memset(buf, 0, MSG_LEN);
+#ifdef __STDC_LIB_EXT1__
+    memset_s(buf, sizeof(*buf), 0, MSG_LEN);
```

Comment:
>sizeof(*buf) [](http://example.com/codeflow?start=18&length=12)

`sizeof(*buf)` is same as `sizeof(buf[0])`, I believe this is not what you intend.

Reply:
All fixed. Could you check again?